addons/medical_vitals/functions/fnc_updateOxygen.sqf

```diff
@@ -0,0 +1,83 @@
+#include "script_component.hpp"
+/*
+ * Author: Brett Mayson
+ * Update the oxygen levels
+ *
+ * Arguments:
+ * 0: The Unit <OBJECT>
```

Comment:
```suggestion
 * 0: The unit <OBJECT>
```

Reply:
I'm leaving these to match the existing files